src/other/Table.js

```diff
@@ -17,6 +18,7 @@
         this._selectionPrevClick = null;
         this._paginatorTableSpacing = 4;
     }
+    Table.prototype = Object.create(Widget.prototype);
     Table.prototype = Object.create(HTMLWidget.prototype);
```

Comment:
Just to elaborate:
```
Table.prototype = Object.create(HTMLWidget.prototype);
```
Is basically saying - let Table.prototype equal HTMLWidget.prototype.  The effect of this is that TAble now has all the same methods that HTMLWidget had defined for itself (which in turn included all the Widget.prototype methods).